contrib/duckdb_fdw/Trunk.toml

```
@@ -0,0 +1,21 @@
+[extension]
+name = "duckdb_fdw"
+version = "0.0.1"
+repository = "https://github.com/alitrack/duckdb_fdw"
+license = "MIT"
+description = "This PostgreSQL extension implements a Foreign Data Wrapper (FDW) for DuckDB."
+homepage = "http://alitrack.com/"
+documentation = "https://github.com/alitrack/duckdb_fdw"
+categories = ['connectors']
+
+[build]
+postgres_version = "15"
+platform = "linux/amd64"
+dockerfile = "Dockerfile"
+install_command = """
+    cd duckdb_fdw && make install USE_PGXS=1
+    set -xe
+    cp libduckdb.so /usr/lib/postgresql/15/lib/
+    cp libduckdb.so $(pg_config --libdir)
```

```suggestion
    cp libduckdb.so $(pg_config --libdir)/duckdb_fdw/
```

@vrmiguel did you mean like this?